### DISCUSSION TITLE
Openai API key not working

### DISCUSSION THREAD

**Discussion by AgentAashik** [community/community#55799] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Uncaught TypeError TypeError: Failed to fetch
    at <anonymous> (c:\Users\Siva Baskar\Documents\ChatGPT-API-Integration-in-HTML-CSS-with-JS-PHP-main\script.js:14:28)
<anonymous> @ c:\Users\Siva Baskar\Documents\ChatGPT-API-Integration-in-HTML-CSS-with-JS-PHP-main\script.js:14:28

**Comment by jge162** [comment]
Maybe you have exceeded your rate limit? If it worked before then issue probably lies with your subscription perhaps.

**Reply by AgentAashik** [reply]
I checked that too

**Reply by jge162** [reply]
Did it work before though? Or you are trying to setup. its hard to debug your scenario, with the amount of info in your post.

**Reply by AgentAashik** [reply]
I'm trying to setup

**Reply by jge162** [reply]
The endpoint you are using is correct? you double checked it.

**Reply by AgentAashik** [reply]
Can you check for me I'm just learning now Php
[ChatGPT-API-Integration-in-HTML-CSS-with-JS-PHP-ma.zip](https://github.com/community/community/files/11503684/ChatGPT-API-Integration-in-HTML-CSS-with-JS-PHP-ma.zip)

**Reply by AgentAashik** [reply]
I doing my mini project in my college
I just want a AI Chatbot related like GPT that's why I'm doing this

**Reply by jge162** [reply]
I cannot click on the link sorry. But, check the instructions on API endpoint for the OpenAI API website.

**Reply by jge162** [reply]
> I doing my mini project in my college I just want a AI Chatbot related like GPT that's why I'm doing this

nice good luck sounds like you are just missing some steps from getting it to work.

**Reply by AgentAashik** [reply]
Do you have any project related to this

**Reply by jge162** [reply]
No I do not use openAI on anything cause it is not free. Sorry.

**Reply by AgentAashik** [reply]
Do you know any free API key where available

**Reply by jge162** [reply]
No idea sorry

**Reply by AgentAashik** [reply]
Ok thanks for the help
